### DISCUSSION TITLE
[FEEDBACK] On ship audio/sound effects.

### DISCUSSION THREAD

**Discussion by Servomotor** [tsunamayo/Starship-EVO#3971] [Suggestions]
I love the ship sound effects and audio. I love the start up sound when you get into the yoke and the general movement sound when flying around. 

I know you have a lot of core/foundational work to do with the new stretch tech/voxel elimination/migration, planets, etc. But I would love to see a way to give ships sound bricks/blocks. Basically a vehicle effects brick that we can select either a number of presets (imagine voice selection for characters in games like Elder Scrolls Online), or a some sliders to tune our ships sound. 

I have never seen audio customization done and I think it could help Starship Evo to stand apart from the competition, on top of the already stellar building capabilities. I dont know if there would be a performance impact (i imagine not, just would increase the file size on our PC's). 

Thank you again.

**Comment by BigBadKangaroo** [comment]
100% agree on that. Having the option to give our ships a uniqe sound would be amazing! 
Doesn't need to be an audio mixer typ of thing, presets would do the job just fine for that.

**Comment by tsunamayo** [comment]
Hi, it would be amazing indeed but honestly I will never have time for this... I think I have done enough to stand out from the competition building wise, I need gameplay...
Cheers

**Comment by PFSCommissar** [comment]
Sorry to necro a thread but i see it as preferaBle to starting a new topic…

I was thinking about this today, and how in SE there would be ambient sound effects when you walked in a ship. It added a whole level of immersion. As someone who used to design game levels, i can say that sound is often underestimated in how much it brings to the table. 

I think the easiest way to do it is just have sound effects play when you’re in a gravity field. If you want to take a step further, give the player the ability to choose whether a field makes a sound, or even which track to play. 

I know you want to focus on gameplay, but consider this when you’re more satisfied with other things. It can really add a lot.

**Reply by tsunamayo** [reply]
@PFSCommissar Yes I agree I wanted to work on that at one point. I wanted submarine like type of sounds, for larger ship at least. And also have the logic gate do some random subtle beeps, and have reactor and other system do a subtle mumble. But that is low priority as it require some SFX sourcing and some tech.
Btw I am about to test your module thanks a lot for them! You can test the PG algo yourself in the new build btw.

**Comment by kynamats** [comment]
On a related note with this as I've also been thinking about it - I've noticed projectiles have doppler shift on their sfx, which means the tech for live pitch changing is in there somewhere - this and volume options on current sfx via some sort of sound options interface on interactable blocks would add a whole load of options that players could use to make sounds more immersive with potentially less work to you.
